docs/site/todo-list-tutorial-model.md

```diff
@@ -68,10 +68,15 @@ Model TodoList was created in src/models/
 ```
 
 Now that we have our new model, we need to define its relation with the `Todo`
-model. Add the following property to the `TodoList` model:
+model. Add the following import statement and property to the `TodoList` model:
 
 #### src/models/todo-list.model.ts
 
+```ts
+import {hasMany} from '@loopback/repository';
+import {Todo} from './todo.model';
+```
+
 ```ts
```

Comment:
Please remove the lines 78 and 80 to merge the new code block with the old code block into a single block.